### ISSUE TITLE
 alert   could not save file  when write  new  file with different   driver path  

### ISSUE THREAD

**Issue by tablecell** [Androthi/SavageEd#8]
save file only support  current  driver path 
```bat

c: 
savageEd.exe   newfile.txt   # or savageEd.exe   c:\users\newfile.txt
   input text 
ctrl+s 
```
 it's ok 

but  write to file with different   driver path  alert   could not save file 
```bat

c: 
savageEd.exe   d:\newfile.txt 
   input text 
ctrl+s 
```
alert   could not save file

**Comment by Androthi** [comment]
Thanks,  I'll try to get a hot fix in the next couple of days.

**Comment by Androthi** [comment]
fixed in v0.10.18 prerelease
